adapter/config/default_config.go

```diff
@@ -197,6 +197,8 @@ var defaultConfig = &Config{
 					UseForSigning:         true,
 				},
 			},
+			JwksRatelimitQuota:               1000,
+			JwksRatelimitTimeWindowInSeconds: 10,
```

Comment:
Shall we include these in, config TOML template file

Reply:
fixed